device/pcie/pci_device.hpp

```diff
@@ -56,6 +59,9 @@ struct PciDeviceInfo
     // onto this struct as methods?  e.g. current_link_width etc.
 };
 
+// Do we want to put everything into this file into tt::umd namespace?
+using tt::umd::semver_t;
```

Comment:
This seems like an overkill
In any case, yes, we do want everything to leave inside tt::umd eventually.

Reply:
Can you clarify?

Reply:
P.S. comment has a typo, it should read "Do we want to put everything _in_ this file into tt::umd namespace?"